Chrome Extension Files/js/inject.js

```diff
@@ -1,40 +1,4 @@
-chrome.storage.sync.get(function(data) {
```

Comment:
hmmm it appears as though I accidentally wrote over this file, but it would've been deprecated by my other changes, regardless.